### DISCUSSION TITLE
How to manage sync rule of one of the accounts?

### DISCUSSION THREAD

**Discussion by earthmoon** [bpozdena/OneDriveGUI#77] [Q&A]
Hello,
I have used LInux for several day, so I can't understand the Linux command well. As in the [site](https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#performing-a-selective-sync-via-sync_list-file) instructes, I failed to sync one selected folder. 
I added two Onedrive accouts. I only want to sync one folder named "zzz" on the second account and all folder on the first account. So, entered into `~/.config/onedrive/accounts/theSecondAccountName`, I created a new file named `sync_list`. Then I opened this file and add a line of code of `zzz`. 
Then I started the Onedrive GUI, then clicked the lower middle button to open the account setting, then I selected the second account, then I chose the `Selective Sync` tab. Then I clicked the `Sync root files` choice. Then I clicked the save button.  Then in the OnedriveGUI, I clicked sync, but it still sync all the files of the second account. 
Anything I did was wrong. May you help me? 
Thanks.

**Comment by bpozdena** [comment]
Hi @earthmoon ,

this does not sound like a GUI specific issue, but if you need to sync only `zzz` directory, your config should look something like this:
![image](https://user-images.githubusercontent.com/24818591/210518395-09296412-bf44-4329-afbf-714c3b40ddeb.png)

Note the directory names are case sensitive. 

Once you have saved the change, you will need to stop and start the sync again. You will be asked to perform a full re-sync.

**Reply by earthmoon** [reply]
Thanks for your help.  What is the `sync_list`, which I saw in the [site](https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#performing-a-selective-sync-via-sync_list-file)?
Can I set it as: `/zzz` or `/zzz/` ?
Should I enable the **Sync root files**?
Thanks.

**Reply by bpozdena** [reply]
>What is the sync_list, which I saw in the [site](https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#performing-a-selective-sync-via-sync_list-file)?

The `Selective Sync` tab simply allows you to edit the `sync_list` file. You can just use the GUI as it's the whole point of it. No need to edit any files manually. 

> Can I set it as: /zzz or /zzz/ ?

Both options should have the same outcome (based on the documentation). I've only tested it with `/zzz/*` .


> Should I enable the Sync root files?

You should enable this only if you have some files in the root of sync directory (default: `~/OneDrive/`) . It has no effect on directories.

**Reply by earthmoon** [reply]
> > What is the sync_list, which I saw in the [site](https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#performing-a-selective-sync-via-sync_list-file)?
> 
> The `Selective Sync` tab simply allows you to edit the `sync_list` file. You can just use the GUI as it's the whole point of it. No need to edit any files manually.
> 
> > Can I set it as: /zzz or /zzz/ ?
> 
> Both options should have the same outcome (based on the documentation). I've only tested it with `/zzz/*` .
> 
> > Should I enable the Sync root files?
> 
> You should enable this only if you have some files in the root of sync directory (default: `~/OneDrive/`) . It has no effect on directories.

Hello,
I have tested `/zzz`, `/zzz/`, and `/zzz/*`. `/zzz/` and `/zzz/*` is the same, so they will sync the **zzz** folder. However, `/zzz` will copy all the file into the folder **zzz**. As I have tested the code `/zzz`, there are so many folders and files in the **zzz** folder.
I want to request another help. How to delete some local files but keep them in the cloud?
Thanks.

**Reply by bpozdena** [reply]
> How to delete some local files but keep them in the cloud?

Once again, question specific to the client, not GUI.

Anyway, you can just add the path of the file into the selective sync and exclude it there with prepended exclamation mark. 
Or just add the file name into Skip File list under Excluded Files tab. 

Ensure to first test with files that are not important to you.

**Reply by earthmoon** [reply]
> > How to delete some local files but keep them in the cloud?
> 
> Once again, question specific to the client, not GUI.
> 
> Anyway, you can just add the path of the file into the selective sync and exclude it there with prepended exclamation mark. Or just add the file name into Skip File list under Excluded Files tab.
> 
> Ensure to first test with files that are not important to you.

Thanks for your kind instruction. I tested and found that I can delete any files those are not in the **Sync list**, and they would not be deleted in the cloud. As there is something in the **Sync list**, it will obey the rule in the **Sync list** and other files is just like local files.
Thanks again.

**Comment by earthmoon** [comment]
Thanks a lot. I know now.
________________________________
